.github/actions/custom-actions/aselo_development_custom/action.yml

```diff
@@ -95,7 +95,11 @@ runs:
       with:
         ssm_parameter: "/development/line/${{inputs.account-sid}}/channel_access_token"
         env_variable_name: "LINE_CHANNEL_ACCESS_TOKEN"
-    # Line environment variables
+    - name: Set helpline Line Studio Flow SID
+      uses: "marvinpinto/action-inject-ssm-secrets@latest"
+      with:
+        ssm_parameter: "/development/twilio/${{inputs.account-sid}}/line_studio_flow_sid"
+        env_variable_name: "LINE_STUDIO_FLOW_SID"
```

Comment:
I was not expecting to see any Line variable in this PR.
Is it missing merging `master` into here, or you're really including a missing variable?

Reply:
It was something that was missed in the initial Line conversion because I set the variable up manually in the serverless environment whilst doing the initial dev